src/server/server_family.cc

```diff
@@ -55,6 +55,9 @@ ABSL_FLAG(string, dbfilename, "dump-{timestamp}", "the filename to save/load the
 ABSL_FLAG(string, requirepass, "",
           "password for AUTH authentication. "
           "If empty can also be set with DFLY_PASSWORD environment variable.");
+ABSL_FLAG(bool, admin_nopass, false,
```

Comment:
you can move this flag to main_service.cc so that `RequireAdminAuth` will be local to that file.

Reply:
Wanted to keep it here, near to requirepass, to bundle it together as security related flags, but as you wish !